scripts/tag-and-release.ts

```diff
@@ -1,7 +1,9 @@
+import { join } from 'node:path';
+
 import { tagAndRelease } from '../src/release-scripts';
 
 const main = async () => {
-  await tagAndRelease('commons');
+  await tagAndRelease('commons', join(__dirname, '../package.json'));
```

Comment:
I suppose another, maybe more flexible, option could be to just load the `package.json` in this script and pass the version string in, rather than the path and let `tagAndRelease` load it.

That said, I think this is fine and will force more consistency